python/lar_constraints.py

```diff
@@ -1149,3 +1150,119 @@ def v705_const(self, row):
 			row["aus_1"] = "6"
 			row["aus_result_1"] = "17"
 		return row
+	
+	def v709_const(self, row):
+		"""1) If Street Address, City, and Zip Code is reported Exempt, then all three
+			 must be reported Exempt."""
+		if row["street_address"] == "Exempt" and row["city"] == "Exempt" and row["zip_code"] == "Exempt":
+			row["street_address"] = "Exempt"
+			row["city"] = "Exempt"
+			row["zip_code"] = "Exempt"
+		return row
+	
+	def v710_const(self, row):
+		"""1) Credit Score of Applicant or Borrower, Credit
```

Comment:
for clarity it may be good to add the qualifier: If the Credit Score exemption election is taken,